.github/workflows/notify_on_broken_link_detected.yaml

```
@@ -0,0 +1,23 @@
+name: notify on broken link detected
+on:
+  workflow_dispatch:
+
+env:
+  HOST: https://osscameroon.com,https://blog.osscameroon.com
```

```suggestion
  HOST: https://osscameroon.com,https://blog.osscameroon.com,https://camerapps.com,https://miniyotas.osscameroon.com,http://devsurvey.osscameroon.com,https://www.prolanghistory.com
```
🙏🏽 

This make me think... it could be grat to have an option for checking subdomains, can you create an issue for that too @pythonbrad please 🙏🏽 ?
And also... i think, it could be great to have this list of link in a file BUT (not the scope of this Pull Request), and am open to options here, what do you think about this idea @pythonbrad  ?